recipes/ghc-bootstrap/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "ghc-bootstrap" %}
+{% set version = "8.2.2" %}
+{% set unixsha256 = "cd7afbca54edf9890da9f432c63366556246c85c1198e40c99df5af01c555834" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://downloads.haskell.org/~ghc/{{ version }}/ghc-{{ version }}-x86_64-deb7-linux.tar.xz  # [linux]
+    sha256: {{ unixsha256 }}  # [linux]
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - binutils_impl_linux-64  # [linux and c_compiler != 'toolchain_c' ]
+    - binutils  # [linux and c_compiler == 'toolchain_c' ]
+    - gmp
+  host:
+    - xz
+    - binutils_impl_linux-64  # [linux and c_compiler != 'toolchain_c' ]
+    - binutils  # [linux and c_compiler == 'toolchain_c' ]
+    - gmp
+
+test:
+  commands:
+    - ghc --help
```

Comment:
By moving the compilation test to run_test.sh or within commands you will also need to make sure you put the {{ compiler('c') }} as part of the requirements section.

Reply:
added running the run_test.sh and changed the dependencies accordingly